packages/frontend/src/components/connect-wallet/index.tsx

```diff
@@ -116,7 +116,7 @@ export const ConnectWallet = () => {
                     ref={accountPopoverRef}
                 />
             )}
-            <div className="flex flex-col gap-4 xl:flex-row">
+            <div className="flex gap-4 flex-row">
```

Comment:
```suggestion
            <div className="flex gap-4">
```

`flex-row` is default so it's not needed.